### PR TITLE
Add tracking to contract overview

### PR DESCRIPTION
**Pull request by shift4id**

### CODE REVIEW

**Comment by jnsdls** [comment]
#### 📝 Changed routes:


| [**/solutions/minting**](https://thirdweb-o8892kh37.thirdweb-preview.com/solutions/minting) |
|:-:|
|<a href="https://thirdweb-o8892kh37.thirdweb-preview.com/solutions/minting">Screenshot of /solutions/minting</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/-N1VQ7bnYVByxt2QkJlU3WC2ur1XapWU6lRq7KB6cLQ9U0Eqcl7xFGNoXFfCwZjS0_KyUFpgrQq6o-oNkshwsg.png">(view full size)</a> |







Commit 8d5735d4406eceaee61f9883fc138341cea35f40 (https://thirdweb-o8892kh37.thirdweb-preview.com).

**Review by nachoiacovino — APPROVED** [review]
lgtm!

File: constants/mappings.ts [hunk 1]
```diff
@@ -414,7 +414,7 @@ export const CHAIN_ID_TO_GNOSIS = Object.entries(GNOSIS_TO_CHAIN_ID).reduce(
     [chainId]: gnosis,
   }),
   {} as Record<
-    typeof GNOSIS_TO_CHAIN_ID[keyof typeof GNOSIS_TO_CHAIN_ID],
+    (typeof GNOSIS_TO_CHAIN_ID)[keyof typeof GNOSIS_TO_CHAIN_ID],
```
Comment (nachoiacovino) [inline constants/mappings.ts:417]: why this change tho

Comment (jnsdls) [reply constants/mappings.ts:417]: think it's autoformat
